terraform/ec2/main.tf

```diff
@@ -78,6 +78,7 @@ locals {
 
   # get SSM package version, latest is the default version
   ssm_package_version = var.aoc_version == "latest" ? "\"\"" : var.aoc_version
+  testcase_name       = split("/", var.testcase)[2]
```

Comment:
Is there a way to get the last element in list in terraform?

Reply:
Something like element(split("/", var.testcase), length(split("/", var.testcase))-1) should work. However the same statement is used to get testcase name by performance and soaking tests. Prefer to refactoring all of them in separate PR.